generators/app/index.js

```diff
@@ -128,35 +134,48 @@ class ConsoleGenerator extends Generator {
 
     spinner.start()
 
-    spinner.text = 'Getting projects...'
+    spinner.text = 'Getting Projects...'
     const projects = (await this.sdkClient.getProjectsForOrg(orgId)).body
     spinner.stop()
 
-    const projectsList = projects.map(item => item.title)
+    // show projects by title and reverse order to show latest first
+    const projectsList = projects.map(item => item.title).reverse()
     const promptFunc = this.allowCreate ? this.customPrompt.promptSelectOrCreate : this.customPrompt.promptSelect
     const projectResult = await promptFunc('Project', projectsList)
     let project = projects.find(item => item.title === projectResult)
 
     if (!project) { // create new
       console.log('Enter Project details:')
       const name = await this.customPrompt.promptInput('Name', {
-        default: projectResult,
-        validate: validateName
+        validate: validateProjectName
       })
       const title = await this.customPrompt.promptInput('Title', {
-        default: projectResult,
-        validate: validateTitle
+        validate: validateProjectTitle
+      })
+      const description = await this.customPrompt.promptInput('Description', {
+        validate: validateProjectDescription,
+        default: ''
       })
-      const description = await this.customPrompt.promptInput('Description', { default: '' })
 
       spinner.start()
 
-      spinner.text = 'Creating project...'
+      // create the projectId
+      spinner.text = 'Creating Project...'
       const createdProject = (await this.sdkClient.createProject(orgId, { name, title, description, type: this.projectType })).body
+      const projectId = createdProject.projectId
+
+      // create the missing stage workspace
+      spinner.text = 'Creating Stage Workspace...'
+      await this.sdkClient.createWorkspace(orgId, projectId, { name: 'Stage' })
+
+      // enable runtime on the Production and Stage workspace
+      spinner.text = 'Enabling Adobe I/O Runtime...'
+      const workspaces = (await (this.sdkClient.getWorkspacesForProject(orgId, projectId))).body
+      await Promise.all(workspaces.map(w => this.sdkClient.createRuntimeNamespace(orgId, projectId, w.id)))
```

Comment:
This is a comment for everything in the project and workspace creation. What happens when there is an exception somewhere, there could be a project created half-baked, and we don't delete it, correct? 

We don't have a concept of "transactions" for creating a project, perhaps something to keep in mind for the future, or suggest that all this be handled in the Console API for the future.

Reply:
very good point, yes this is correct
will try to fix that, would be good to suggest it to the Console API too

Reply:
we can push this off as a new feature

Reply:
ok, + I started working on this and realized deleteRuntimeNamespace doesn't exist yet so we can't really cleanup things fully. Will open an issue after we merge this